### PR TITLE
Add the remaining keyboard shortcuts from the terminal to the Mod4+r prompt.

### PR DESCRIPTION
**Pull request by b80905**
Not sure why Ctrl+p, Ctrl+n, Alt+b, Alt+f, Alt+d, Alt+BackSpace were not added along with Ctrl+k, Ctrl+a, etc.

### CODE REVIEW

**Comment by Elv13** [comment]
+1, look fine to me, if nobody else have anything to say, I will merge this

**Comment by Elv13** [comment]
last call

File: lib/awful/prompt.lua [hunk 1]
```diff
@@ -64,6 +64,42 @@ local function history_check_load(id, max)
     end
 end
 
+local function is_word_char(c)
```
Comment (blueyed) [inline lib/awful/prompt.lua]: This should be moved to `awful.util` - and there might be something like that maybe already..

**Comment by blueyed** [comment]
Cool.  Reminds me of a similar PR of mine (https://github.com/awesomeWM/awesome/pull/187).

I've not tested it, but can do so later and then merge it - please assign it to me then.
I'm fine also with just merging it, if you have tested it at least slightly.

**Comment by blueyed** [comment]
Merged.
Thanks @b80905!

**Pull request closed without merging**